library/x509_crt.c

```diff
@@ -91,9 +91,8 @@ typedef struct {
  */
 #define X509_MAX_VERIFY_CHAIN_SIZE    ( MBEDTLS_X509_MAX_INTERMEDIATE_CA + 2 )
 
-/*
- * Default profile
- */
+/* Default profile. Do not remove items unless there are serious security
+ * concerns. */
```

Comment:
We don't want to backport the addition of #if defined(MBEDTLS_ECP_C) ?
```C
#if defined(MBEDTLS_ECP_C)
0xffffffff,
#else
0,
#endif
```

Reply:
The value isn't used when `MBEDTLS_ECP_C` is not defined, so I don't think it matters. Seeing the inconsistency now, I kind of regret the extra complexity of the `#ifdef` in 3.0 but I don't think it matters.

Reply:
fine by me.